server/public/model/limits.go

```diff
@@ -4,6 +4,7 @@
 package model
 
 type UserLimits struct {
-	MaxUsersLimit   int64 `json:"maxUsersLimit"`   // max number of users allowed
-	ActiveUserCount int64 `json:"activeUserCount"` // actual number of active users on server. Active = non deleted
+	MaxUsersLimit     int64 `json:"maxUsersLimit"`     // soft limit for max number of users.
+	MaxUsersHardLimit int64 `json:"maxUsersHardLimit"` // hard limit for max number of active users.
+	ActiveUserCount   int64 `json:"activeUserCount"`   // actual number of active users on server. Active = non deleted
```

Comment:
we will need to add the new field to the types of webapp, will that be a separate effort?

Reply:
I would see it as a separate effort if we ever need to check for hard limit in the webapp. So far this is only checked in the server

Reply:
We don't need the hard limit in webapp for now so its fine. The old value is unchanged and works fine.